R/betydb.R

```diff
@@ -186,14 +139,11 @@ betydb_auth <- function(x,y,z){
   }
 }
 
-makeurl <- function(x, fmt){
-  fmt <- match.arg(fmt, c("json","xml","csv"))
-  paste0(betyurl(), paste0(x, "."), fmt)
-}
-
-makeidurl <- function(x, id, fmt){
+makeurl <- function(x, id, fmt){
```

Comment:
Will we still be querying by a single `id` or can this argument be removed from all functions here?